Networking15.02Escaping/Networking15.02Escaping/CacheService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CacheService.swift
+//  Networking15.02Escaping
+//
+//  Created by Tania Harbarchuk on 05.09.2023.
+//
+
+import UIKit
+
+class CacheService {
+    
+    static let shared = CacheService()
+    
+    //MARK: - Private Property
+    private var cache = NSCache<NSString, UIImage>()
+    private var elementCount = 0
+    private var maxElementCount = 100
+    
+    //MARK: - Public Func
+    func saveCachedImage(image: UIImage, url: NSString) {
```

Comment:
why you put force unwrapped UIImage and NSString but not UIImage? and String, and unwrap it here and downcast to NSString here ??